test/test_mypy.py

```diff
@@ -68,6 +84,8 @@ class TestMypyFails(unittest.TestCase):
     def ensure_mypy_fails(self, filename: str) -> None:
         if api is None:
             raise unittest.SkipTest("Mypy is not installed")
+        if TypedDict is None:
+            raise unittest.SkipTest("TypedDict is supported on Python 3.8+ only")
```

Comment:
This seems redundant given that typeddict_client is already skipped in test_mypy_failures. Also this skip is overeager and will skip all the files on 3.7 which we don't want.